### PR TITLE
New Concatenation Operator `||`

### PR DESCRIPTION
**Pull request by gem-neo4j**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-921.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-921.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)
- [Operators](https://neo4j-docs-cypher-921.surge.sh/cypher-manual/5/syntax/operators)

**Comment by gem-neo4j** [comment]
[GQL compliance - concatenation operator](https://trello.com/c/z392Upso/925-gql-compliance-concatenation-operator)

**Review by loveleif — APPROVED** [review]

File: modules/ROOT/pages/syntax/operators.adoc [hunk 1]
```diff
@@ -562,6 +562,23 @@ RETURN 'neo' + '4j' AS result
 1+d|Rows: 1
 |===
 
+[[syntax-concatenating-two-strings-doublebar]]
+=== Concatenating two `STRING` values with `||`
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/syntax/operators.adoc:567]:
```suggestion
[role=label--new-5.19]
[[syntax-concatenating-two-strings-doublebar]]
=== Concatenating two `STRING` values with `||`
```

File: modules/ROOT/pages/syntax/operators.adoc [hunk 2]
```diff
@@ -916,6 +933,23 @@ RETURN [1,2,3,4,5] + [6,7] AS myList
 1+d|Rows: 1
 |===
 
+[[syntax-concatenating-two-lists-with-doublebar]]
+=== Concatenating two lists using `||`
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/syntax/operators.adoc:936]:
```suggestion
[role=label--new-5.19]
[[syntax-concatenating-two-lists-with-doublebar]]
=== Concatenating two lists using `||`
```

File: modules/ROOT/pages/syntax/operators.adoc [hunk 3]
```diff
@@ -562,6 +562,23 @@ RETURN 'neo' + '4j' AS result
...
+[source, cypher]
+----
+RETURN 'neo' || '4j' AS result
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+|===
+| +result+
+| +"neo4j"+
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/syntax/operators.adoc]: Plus sign is not necessary when cols=m is added.

```suggestion
| result
| "neo4j"
```

File: modules/ROOT/pages/syntax/operators.adoc [hunk 4]
```diff
@@ -916,6 +933,23 @@ RETURN [1,2,3,4,5] + [6,7] AS myList
...
+[source, cypher]
+----
+RETURN [1,2,3,4,5] || [6,7] AS myList
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+|===
+| +myList+
+| +[1,2,3,4,5,6,7]+
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/syntax/operators.adoc]:
```suggestion
| myList
| [1,2,3,4,5,6,7]
```

**Review by JPryce-Aklundh — APPROVED** [review]
Thanks @gem-neo4j!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-921.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-921.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)
- [Operators](https://neo4j-docs-cypher-921.surge.sh/cypher-manual/5/syntax/operators)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-921.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-921.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)
- [Operators](https://neo4j-docs-cypher-921.surge.sh/cypher-manual/5/syntax/operators)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**